pkgs/development/tools/misc/ccache/default.nix

```diff
@@ -15,43 +15,46 @@ let ccache = stdenv.mkDerivation rec {
     substituteInPlace Makefile.in --replace 'objs) $(extra_libs)' 'objs)'
   '';
 
-  doCheck = true;
+  doCheck = !stdenv.isDarwin;
```

Comment:
We frequently see issues with tests on darwin because `$TMPDIR` is set to something long (usually related to unix sockets) perhaps this helps.

```
preCheck = ''
  export TMPDIR=/tmp
'';
```